catalog/core/catalog-core-migratable/src/main/java/org/codice/ddf/catalog/migratable/impl/CatalogMigratableConfig.java

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.migratable.impl;
+
+import java.nio.file.Path;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Contains configuration constants used to change the behavior of catalog migratable operations.
+ */
+public class CatalogMigratableConfig {
+
+    private static final int MAX_CARDS_PER_FILE = 50000;
+
+    private static final int MAX_QUERY_PAGE_SIZE = 100000;
+
+    private static final int MIN_QUERY_PAGE_SIZE = 1;
+
+    private static final int MAX_THREADS = 128;
+
+    private static final String FILE_PREFIX = "catalogExport";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CatalogMigratableConfig.class);
+
+    private int exportCardsPerFile;
+
+    private Path exportPath;
+
+    private int exportQueryPageSize;
+
+    private int exportThreadCount;
+
+    public CatalogMigratableConfig() {
+        exportPath = null;
+        this.exportCardsPerFile = 1;
+        this.exportQueryPageSize = 5000;
+        this.exportThreadCount = 8;
+    }
+
+    public Path getExportPath() {
+        return exportPath;
+    }
+
+    public void setExportPath(Path exportPath) {
+        this.exportPath = exportPath;
+    }
+
+    public int getExportCardsPerFile() {
+        return exportCardsPerFile;
+    }
+
+    public void setExportCardsPerFile(int exportCardsPerFile) {
+        if (exportCardsPerFile >= 1 && exportCardsPerFile <= MAX_CARDS_PER_FILE) {
```

Comment:
If you invert this condition you can reduce the cyclomatic complexity of this method.